src/Servers/Kestrel/Core/src/Internal/Http/HttpHeaders.Generated.cs

```diff
@@ -102,6 +102,215 @@ internal enum KnownHeaderType
         WWWAuthenticate,
     }
 
+    internal static class HttpHeadersCompression
+    {
+        internal static (int index, bool matchedValue) MatchKnownHeaderQPack(KnownHeaderType knownHeader, string value)
+        {
+            switch (knownHeader)
+            {
+                case KnownHeaderType.Age:
+                    switch (value)
+                    {
+                        case "0":
+                            return (2, true);
+                        default:
+                            return (2, false);
+                    }
+                case KnownHeaderType.ContentLength:
+                    switch (value)
+                    {
+                        case "0":
+                            return (4, true);
+                        default:
+                            return (4, false);
+                    }
+                case KnownHeaderType.Date:
+                    switch (value)
+                    {
+                        case "":
+                            return (6, true);
+                        default:
+                            return (6, false);
+                    }
```

Comment:
Thoughts on skipping the switch if the only option is `""`?

Reply:
I'm inclined to say it isn't worth checking if the only option for the value is `""`. Do we have a good way to benchmark this?

Reply:
There are microbenchmarks for this code. I changing this won't have any impact on perf.

I'm going to remove `""` because it isn't here because it is a common default value. It is here just because there isn't a common default value, and so was left blank.